pkg/kubelet/cm/node_container_manager_linux.go

```diff
@@ -183,9 +187,37 @@ func getCgroupConfig(rl v1.ResourceList) *ResourceConfig {
 	}
 	rc.HugePageLimit = HugePageLimits(rl)
 
+	// In the case of a None policy, cgroupv2 and systemd cgroup manager, we must make sure systemd is aware of the cpuset cgroup.
+	// By default, systemd will not create it, as we've not chosen to delegate it, and we haven't included it in the Apply() request.
+	// However, this causes a bug where kubelet restarts unnecessarily (cpuset cgroup is created in the cgroupfs, but systemd
+	// doesn't know about it and deletes it, and then kubelet doesn't continue because the cgroup isn't configured as expected).
+	// An alternative is to delegate the `cpuset` cgroup to the kubelet, but that would require some plumbing in libcontainer,
```

Comment:
Just curious, what plumbing do you refer to here? IOW what's missing?

Reply:
we currently use libcontainer for all of our cgroup management and there's no way to set a systemd property through the libcontainer manager. we could just use a godbus instance ourselves but it'd take some setup and copied code